src/components/Comments/ReplyCommentItem.js

```diff
@@ -272,40 +306,69 @@ const ReplyCommentItem = ({
-              style={styles.btnReply}
-              onPress={onPress}>
-              <MemoCommentReply />
-              <Text style={styles.btnReplyText}>Reply</Text>
-            </ButtonHightlight>
-          </TouchableOpacity>
+          <>
+            <TouchableOpacity testID="replyBtn" activeOpacity={1} onPress={onPress}>
+              <ButtonHightlight
+                onLongPress={handleLongPress}
+                style={[
+                  styles.btnReply,
+                  {
+                    marginRight: 0,
+                    paddingRight: 0
+                  }
+                ]}
+                onPress={onPress}>
+                <MemoCommentReply />
+              </ButtonHightlight>
+            </TouchableOpacity>
+            {!comment.is_you && (
+              <TouchableOpacity onPress={onPressDm} testID="sendDMbtn" activeOpacity={1}>
+                <ButtonHightlight
+                  onLongPress={handleLongPress}
+                  onPress={onPressDm}
+                  style={[
+                    styles.btnBlock(comment.user.id === yourselfId),
+                    styles.btn,
+                    {
+                      marginRight: 0,
+                      paddingRight: 0
+                    }
+                  ]}>
+                  <MemoSendDM />
+                </ButtonHightlight>
+              </TouchableOpacity>
+            )}
+          </>
         )}
         <TouchableOpacity onPress={() => onBlock(comment)} testID="btnBlock" activeOpacity={1}>
           <ButtonHightlight
             onLongPress={handleLongPress}
             onPress={() => onBlock(comment)}
-            style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}>
+            style={[
+              styles.btnBlock(comment.user.id === yourselfId),
+              styles.btn,
+              {
+                marginRight: 16,
+                paddingRight: 0
+              }
+            ]}>
             <IconEn name="block" size={15.02} color={COLORS.balance_gray} />
           </ButtonHightlight>
         </TouchableOpacity>
-        <TouchableOpacity
-          onPress={() => refSheet.current.open()}
-          testID="sendDMbtn"
-          activeOpacity={1}>
-          <ButtonHightlight
-            onLongPress={handleLongPress}
-            onPress={() => refSheet.current.open()}
-            style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}>
-            <MemoSendDM />
-          </ButtonHightlight>
-        </TouchableOpacity>
 
         <TouchableOpacity onPress={onDownVote} testID="downvoteBtn">
           <ButtonHightlight
             onLongPress={handleLongPress}
-            style={[styles.arrowup, styles.btn]}
+            style={[
+              styles.arrowup,
+              styles.btn,
+              {
+                paddingLeft: 0,
+                marginLeft: 0,
+                marginRight: 8,
+                paddingRight: 0
+              }
+            ]}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for displaying the reply and direct message buttons has been updated. Now it checks whether the comment is from the current user or not and displays the direct message button accordingly. This is a necessary change for the new feature being implemented.
```diff
+ <>
+   <TouchableOpacity testID="replyBtn" activeOpacity={1} onPress={onPress}>
+     <ButtonHightlight
+       onLongPress={handleLongPress}
+       style={[
+         styles.btnReply,
+         {
+           marginRight: 0,
+           paddingRight: 0
+         }
+       ]}
+       onPress={onPress}>
+       <MemoCommentReply />
+     </ButtonHightlight>
+   </TouchableOpacity>
+   {!comment.is_you && (
+     <TouchableOpacity onPress={onPressDm} testID="sendDMbtn" activeOpacity={1}>
+       <ButtonHightlight
+         onLongPress={handleLongPress}
+         onPress={onPressDm}
+         style={[
+           styles.btnBlock(comment.user.id === yourselfId),
+           styles.btn,
+           {
+             marginRight: 0,
+             paddingRight: 0
+           }
+         ]}>
+         <MemoSendDM />
+       </ButtonHightlight>
+     </TouchableOpacity>
+   )}
+ </>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->